ci-operator/step-registry/operator/install/operator-install-ref.yaml

```diff
@@ -29,9 +29,6 @@ ref:
     - name: OPERATOR6_CONFIG
       default: ""
       documentation: Refer to OPERATOR1_CONFIG documentation
```

Comment:
why was `PARALLEL` removed? please re-add
should be "false" and the condition in https://github.com/openshift/release/blob/master/ci-operator/step-registry/operator/install/operator-install-commands.sh#L22 should be:
`if [ "${PARALLEL}" = "true" ]`